### PR TITLE
Warnings for big bems.

### PR DESCRIPTION
**Pull request by jaeilepp**
Closes https://github.com/mne-tools/mne-python/issues/3112

### CODE REVIEW

File: mne/bem.py [hunk 1]
```diff
@@ -318,6 +318,9 @@ def make_bem_solution(surfs, verbose=None):
         logger.info('Homogeneous model surface loaded.')
     else:
         raise RuntimeError('Only 1- or 3-layer BEM computations supported')
+    if bem['surfs'][0]['np'] > 10000:
+        warn('The bem surface has %s data points. 5120 should be enough.'
```
Comment (Eric89GXL) [inline mne/bem.py]: Should probably warn that it might not save properly, and to DRY it this can be a `_check_bem_size` private function

Comment (jaeilepp) [inline mne/bem.py]: I suppose that is only a problem with three layer bem. So maybe warn only then.

Comment (jaeilepp) [inline mne/bem.py]: Or maybe it saves to separate files, right? I'll check.

Comment (Eric89GXL) [inline mne/bem.py]: 3-layer BEM gets saved to a single file, just like 1-layer BEM

File: mne/bem.py [hunk 2]
```diff
@@ -318,9 +318,7 @@ def make_bem_solution(surfs, verbose=None):
         logger.info('Homogeneous model surface loaded.')
     else:
         raise RuntimeError('Only 1- or 3-layer BEM computations supported')
-    if bem['surfs'][0]['np'] > 10000:
-        warn('The bem surface has %s data points. 5120 should be enough.'
-             % bem['surfs'][0]['np'])
+    _check_bem_size(bem)
```
Comment (Eric89GXL) [inline mne/bem.py]: to save users some pain, you could also put this warning in `make_bem_model`, the lighter-weight step they likely use before making the solution

File: mne/bem.py [hunk 3]
```diff
@@ -1842,6 +1844,16 @@ def make_flash_bem(subject, overwrite=False, show=True, subjects_dir=None,
     os.chdir(curdir)
 
 
+def _check_bem_size(bem):
+    """Helper for checking bem sizes."""
+    if bem['surfs'][0]['np'] > 10000:
+        msg = ('The bem surface has %s data points. 5120 should be enough.'
```
Comment (Eric89GXL) [inline mne/bem.py]: and actually you might want to say `5120 (ico grade=4) should be enough`

**Comment by jaeilepp** [comment]
Comments addressed.

**Comment by agramfort** [comment]
thx @jaeilepp

**Pull request merged**